app/views/test/components/track_lists/show.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1 Track list
+= form_tag nil, method: :get do
+  = label_tag :state
+  = select_tag :state, options_for_select(%w[Success Error Loading], params[:state])
+  = submit_tag "Submit", name: nil
+
+%br
+Error / Loading states are only seen when filtering
+%hr/
+
+= track_list(@tracks, { endpoint: test_components_track_list_tracks_path, query: { state: params[:state] }, options: { retry: params[:state] == "Loading" } })
```

Comment:
When https://github.com/exercism/v3-website/pull/82 is merged, this needs to be updated.